content/en/profiler/enabling/php.md

```diff
@@ -79,7 +81,9 @@ php hello.php
 Use the `env` directive in the php-fpm’s `www.conf` file, for example:
 
 ```
+; DD_PROFILING_ENABLED is not required for v0.82.0+
```

Comment:
Just want to confirm that this should be a semi-colon and not a # like in the other updates you added.

Reply:
Yes, comments in INI files begin with `;`. Some languages permit `#` but PHP removed this ability at some point (I'm not sure why).